src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -0,0 +1,228 @@
+    PortableInfoboxPageObject info = article.getInfoboxPage();
+    Assertion.assertTrue(info.getBoldElements().size() > 0);
+    Assertion.assertTrue(info.getItalicElements().size() > 0);
+    Assertion.assertTrue(info.getHeaderElements().size() > 0);
+    info.verifyQuotationMarksPresence();
+    info.verifyReferencesPresence();
+    info.verifyImagePresence();
+    info.verifyInfoboxTitlePresence();
+
+  }
+
+  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_002"})
+  public void verifyElementsRedirects() {
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    PortableInfoboxPageObject info = article.getInfoboxPage();
+    //Red link
+    info.clickRedLink(0);
+    info.verifyCreateNewArticleModal();
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    article.getInfoboxPage();
+    //External Link
+    String externalLinkName = info.getExternalLinkRedirectTitle();
+    info.clickExternalLink();
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
+    String externalNavigatedURL = driver.getCurrentUrl();
+    info.compareURLAndExternalLink(externalLinkName, externalNavigatedURL);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    article.getInfoboxPage();
+    //Internal Link
+    String internalLinkName = info.getInternalLinkRedirectTitle(0);
+    info.clickInternalLink(0);
+    try {
+      Thread.sleep(1000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
+    String internalNavigatedURL = driver.getCurrentUrl();
+    info.compareURLAndInternalLink(internalLinkName, internalNavigatedURL);
+  }
+
+  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_003"})
+  public void verifyImagesInWhatLinksHerePage() {
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    PortableInfoboxPageObject info = article.getInfoboxPage();
+    String articleName = article.getArticleName();
+    SpecialWhatLinksHerePageObject links = article.openSpecialWhatLinksHere(wikiURL);
+    links.clickPageInputField();
+    links.typeInfoboxImageName(PageContent.FILE_IMAGE_NAME);
+    links.clickShowbutton();
+    links.verifyInfoboxArticleInList(articleName);
+  }
+
+  @Test(groups = {"PortableInfoboxTests", "PortableInfoboxTests_004"})
+  public void verifyCategoriesInTemplateInvocation() {
+    ArticlePageObject article = new ArticlePageObject(driver);
+    article.openArticleByName(wikiURL, PageContent.PORTABLE_INFOBOX01);
+    PortableInfoboxPageObject info = article.getInfoboxPage();
+    SourceEditModePageObject src = info.navigateToArticleEditPageSrc(wikiURL, PageContent.PORTABLE_INFOBOX_WEBSITE_TEMPLATE);
+    src.focusTextArea();
+    String catName = src.getRandomDigits(9);
+    System.out.println("Random digits: " + catName);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)